src/docs/sdk/features.mdx

```diff
@@ -155,9 +155,28 @@ Include a list of loaded libraries (and versions) when sending an event.
 
 ## Buffer to Disk
 
-Write events to disk before attempting to send, so that they can be retried in the event of a temporary network failure. Needs to implement a cap on the number of stored events.
+This feature is also known as 'Offline Caching'.
 
-This is mostly useful on mobile clients where connectivity is often not available.
+Write events to disk before attempting to send, so that they can be retried in the event of a temporary network failure. Needs to implement a cap on the number of stored events. This is mostly useful on mobile and desktop(e.g: laptop) apps, where stable connectivity is often not available.
```

Comment:
Would it be useful to link to some implementations where this is working? Like to cocao, .NET, electron?

Reply:
Added 4 examples below (linked to the file on the main branch instead of specific lines and a commit sha, so technically could go out of date if the file move or main branch changes but better to point to "latest" I believe